ethereum/spec/src/main/java/tech/pegasys/teku/spec/logic/common/helpers/BeaconStateMutators.java

```diff
@@ -103,6 +104,43 @@ public void initiateValidatorExit(MutableBeaconState state, int index) {
                     exitQueueEpoch.plus(specConfig.getMinValidatorWithdrawabilityDelay())));
   }
 
+  public Optional<UInt64> initiateValidatorExit(
+      MutableBeaconState state,
+      int index,
+      final UInt64 exitQueueEpoch,
+      List<Validator> exitedValidators) {
+    Validator validator = state.getValidators().get(index);
```

Comment:
i would make this variable store state.getValidators, and should only have to call that once hopefully... can improve things a lot the less we call state.get functions...

Reply:
ah.. we can put Validators in the context now.